.github/workflows/spice_mysql_bench_docker.yml

```diff
@@ -8,6 +8,43 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Check disk space
```

Comment:
@Sevenannn  -  could you please elaborate on why we are doing this

Reply:
Because MySQL image build fail due to running out of disk space. These steps prune unused disk space so that MySQL image build can succeed